CHANGES.rst

```diff
@@ -10,9 +10,10 @@ eso
 - Authenticate with ESO using APIs and tokens instead of HTML forms. [#2681]
 - Discontinue usage of old Request Handler for dataset retrieval in favor of new dataportal API. [#2681]
 - Local reimplementation of astroquery's ``_download_file`` to fix some issues and avoid sending a HEAD request
-  just to get the original filename. [#1580]
-- Restore support for .Z files. [#1818]
+  just to get the original filename.
+- Restore support for .Z files.
 - Update tests and documentation.
+- Create test files in temporary directories. [#2903]
```

Comment:
fyi: we don't do changelog for non-user facing changes or non-code changes, anything touching only tests/docs/ci, etc.